### ISSUE TITLE
Assignment 2 MLS

### ISSUE THREAD

**Issue by allenZhangPersonal** [danielepanozzo/gp#7]
Good morning, I just want to confirm that we are solving MLS by picking points from our "3n" points set that are within a certain radius "h" from the current grid point "xi" and calculating the coefficients of our polynomial based on points that are within "h" distance from "xi". Then, we are using the polynomial to evaluate "xi"?

How do we pick the Wendlend radius or do we incorporate the W(x) matrix when approximating the LS? What are sensible radius to pick or we have to experiment to see ourselves?

How can we be certain that when a specific grid point has less than twice the number of polynomial coefficients of points near it, it is a positive outside point instead of an inside point?

Thank you.

**Comment by danielepanozzo** [comment]
Yes, only the points within the support of the Wendland function will have a contribution to the polynomial fit.

You will have to find a radius that works well for each scene through experiments. If you do not have enough points you will have to arbitrarily decide that the point is either inside or outside. This will generate a few spurious small connected components of triangles, which you can filter out at the end. I will discuss this in more detail on Thursday after we complete the marching cube slides.
